data/src/config/server.rs

```diff
@@ -105,6 +107,13 @@ impl Server {
     }
 }
 
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum IdentifySyntax {
+    NickThenPassword,
+    PasswordThenNick,
+}
```

Comment:
Nit: maybe just `NickPassword` / `nick-password`, etc works since it's kebab case?

Reply:
Thanks :D Don't forget to update the docs!

Reply:
Shoot, thanks for the reminder!